tools/go-mod-tidy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Root of repo
+ROOT_DIR=${1:-'..'}
+
+GOMODULE_PATHS=("weaver/core/network/fabric-interop-cc/libs/utils"
```

Comment:
@sandeepnRES I recommend working with path patterns instead of hard-coding paths. Directories get renamed all the time, new go.mod files added/removed and then this script will be brittle, break the CI when these changes are made and the script is not updated.

Reply:
@petermetz I agree, but what do you suggest here, I don't clearly understand how to do that? Should I declare some file in root directory which stores all go module locations and take it from there?

Btw this is a just a helper tool and not used in CI, this is useful when some dependabot PR updates go dependencies and we see `go mod tidy` error in CI tests. If this script is run once, it will fix all those in one go.